packages/cms-lib/links.js

```diff
@@ -0,0 +1,138 @@
+const { getEnv } = require('./lib/config');
+const { ENVIRONMENTS } = require('./lib/constants');
+const { getHubSpotWebsiteOrigin } = require('./lib/urls');
+const { logger } = require('./logger');
+const chalk = require('chalk');
+const { table, getBorderCharacters } = require('table');
+var opn = require('opn');
```

Comment:
This looks to be deprecated in favor of [open](https://www.npmjs.com/package/open), which we're using elsewhere.

Reply:
Ah cool, thanks. I'll change it.